src/core/shared_state/index.ts

```diff
@@ -1,5 +1,4 @@
 export { currentEpisodeAtom } from './currentEpisode';
-export { currentMapPositionAtom } from './currentMapPosition';
```

Comment:
Why don't you reexport everything from `currentMapPosition.ts` here? Import will be shorter

Reply:
It looked okay with v2 atoms, which encapsulate all actions in the same object. With v3, it feels kinda cumbersome and useless to me. We should probably discuss if these index files are even needed for atoms, seems like they haven't been updated in over two years and they don't even include types